docs/docs/sourcing-from-forestry.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Sourcing from Forestry
+---
+
+### Overview
+
+In this guide, we'll walk through setting up a site with content management using [Forestry.io](https://forestry.io/).
```

Comment:
Please do not use "we" and instead "you" throughout the doc
https://www.gatsbyjs.org/contributing/gatsby-style-guide/#use-you-as-the-pronoun